swagger.yaml

```diff
@@ -1829,6 +1857,26 @@ paths:
           description: Unexpected error.
           schema:
             $ref: "#/definitions/Error"
+  /email_confirm/{token}:
```

Comment:
As we introduced versioning for our API, it would be beneficial to move the new endpoint to the 'v1' path.